mappings/net/minecraft/resource/DataPackProfile.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS net/minecraft/class_5359 net/minecraft/resource/DataPackProfile
```

Comment:
more like data pack selection for a server? also not really resource related but more server related, as the lists' element types are like strings